radiometric_normalization/gimage.py

```diff
@@ -173,3 +173,38 @@ def check_comparable(gimages, check_metadata=False):
             raise Exception(
                 'Image {} has different geographic metadata: {} '
                 '(initial: {})'.format(i + 1, image.metadata, metadata))
+
+
+def check_equal(gimages, check_metadata=False):
```

Comment:
Is there a way to name this to reduce potential confusion related to it returning a boolean vs check_comparable which raises an exception?